packages/runtime/datastore/src/test/channelStorageService.spec.ts

```diff
@@ -21,7 +21,7 @@ describe("ChannelStorageService", () => {
                 throw new Error("not implemented");
             },
         };
-        const ss = new ChannelStorageService(tree, storage);
```

Comment:
Consider using TelemetryUTLogger in UTs - I believe it should fail the test if any error event is issueed